src/toil/wdl/wdltoil.py

```diff
@@ -1505,17 +1505,25 @@ def run(self, file_store: AbstractFileStore) -> Promised[WDLBindings]:
 
             # Prepare to use Singularity. We will need plenty of space to
             # download images.
-            if 'SINGULARITY_CACHEDIR' not in os.environ:
-                # Cache Singularity's layers somehwere known to have space, not in home
-                os.environ['SINGULARITY_CACHEDIR'] = os.path.join(file_store.workflow_dir, 'singularity_cache')
+            # Set the Singularity and MiniWDL cache directory to the default Singularity cache directory. This sets the
+            # cache to the same place as Singularity/MiniWDL's default cache directory
+            # With launch-cluster, the singularity and miniwdl cache is set to /var/lib/toil in abstractProvisioner.py
+            # A current limitation with the singularity/miniwdl cache is it cannot check for image updates if the
+            # filename is the same
+            singularity_cache = os.path.join(os.path.expanduser("~"), "/.singularity/cache")
+            miniwdl_cache = os.path.join(os.path.expanduser("~"), "/.cache/miniwdl")
```

Comment:
This sounds like we're actually setting these, but actually these are default fallback values.